### PR TITLE
TT with modified top width

### PR DESCRIPTION
**Pull request by jfernan2**
- TTbar configs with modified top width: 0.7 , 0.85, 1.15 and 1.3 times the nominal top width of 1.31GeV
- Fixes in configs for old Moriond17 Semileptonic hdampUP/Down

### CODE REVIEW

File: bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdamp_widthx0p7_NNPDF31_NNLO_dilepton.input [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+lhans1 306000
+lhans2 306000
+! NNPDF31_nnlo_as_0118
+ebeam1 6500    ! energy of beam 1
+ebeam2 6500    ! energy of beam 2
+qmass  172.5     ! mass of heavy quark in GeV
+facscfact 1    ! factorization scale factor: mufact=muref*facscfact 
+renscfact 1    ! renormalization scale factor: muren=muref*renscfact 
+
+hdamp 237.8775
```
Comment (perrozzi) [inline bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdamp_widthx0p7_NNPDF31_NNLO_dilepton.input]: is hdamp re-fitted or is kept the same?

Comment (jfernan2) [reply bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdamp_widthx0p7_NNPDF31_NNLO_dilepton.input]: It is kept the same as nominal sample, the process and tuning is the same so no need to refit, just a top width change e.g.
https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdamp_NNPDF31_NNLO_dilepton.input#L26
Thanks

**Comment by perrozzi** [comment]
I would suggest to have a representative card for one width and a script to create the others

**Comment by jfernan2** [comment]
But now that the cards have been created, should I revert to your solution? We do not expect any additional width sample to be created/requested.

**Comment by perrozzi** [comment]
ok, fine for me. @efeyazgan can you please crosscheck the cards?

**Pull request merged**